tests/regression/04-mutex/61-allfuns-globs.c

```diff
@@ -0,0 +1,23 @@
+// PARAM: --set allfuns true
```

Comment:
How is this `allfuns` test related?

Reply:
Because function definitions included from headers such as `"/usr/include/x86_64-linux-gnu/bits/byteswap.h" call these functions. I noticed the problem with test `04/32` which is an allfuns test that this one is modelled after.

Reply:
But that test doesn't include the byteswap header: it's not anywhere in the preprocessed output.

Reply:
On my machine it does:

```
./regtest.sh 04 32 --enable justcil > out.cil.c
```

~~~C
#line 108 "/usr/include/x86_64-linux-gnu/bits/byteswap.h"
__inline static __uint64_t __bswap_64(__uint64_t __bsx ) 
{ 
  long tmp ;

  {
#line 111
  tmp = __builtin_bswap64((long )__bsx);
#line 111
  return ((__uint64_t )tmp);
}
}
~~~

Reply:
Hold on, those builtins also declared by CIL: https://github.com/goblint/cil/blob/1e53aa83af4c7d51353092eb1d7ff52979fb20ad/src/cil.ml#L2879-L2891. So all the missing ones (e.g. `bswap128`) should also be added there.